package.json

```diff
@@ -40,6 +40,7 @@
     "build": "lerna run build",
     "check-license": "./scripts/check-license.sh",
     "coverage": "lerna run coverage",
+    "depcheck": "((node scripts/generateDepcheckrc.js --package 'jaeger-ui' && depcheck packages/jaeger-ui && rm -f packages/jaeger-ui/.depcheckrc.json;) || (EXIT_CODE=$?; rm -f packages/jaeger-ui/.depcheckrc.json; exit $EXIT_CODE)); ((node scripts/generateDepcheckrc.js --package 'plexus' && depcheck packages/plexus && rm -f packages/plexus/.depcheckrc.json;) || (EXIT_CODE=$?; rm -f packages/plexus/.depcheckrc.json; exit $EXIT_CODE));",
```

Comment:
could we just put this into a script, similar to ./scripts/check-license.sh? I think trying to fit this into a single line makes it harder than it needs to be, and the syntax is not portable anyway (e.g. for Windows) so why not a bash script.

Reply:
Yes, we can move it to a shell script too. Working on it.

> Syntax is not portable anyway (e.g. for Windows)
This would most probably work on git bash (on Windows, which is a UNIX-based terminal.

Reply:
done 🚀